AlertaDengue/AlertaDengue/settings.py

```diff
@@ -104,14 +115,48 @@ def read_admins(value):
 # https://docs.djangoproject.com/en/1.6/ref/settings/#databases
 
 DATABASES = {
-    'default': config(
-        'DATABASE_URL', default='sqlite:///geodjango.db', cast=db_url
-    )
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': os.getenv('PSQL_DB'),
+        'USER': os.getenv('PSQL_USER'),
+        'PASSWORD': os.getenv('PSQL_PASSWORD'),
+        'HOST': os.getenv('PSQL_HOST'),
+        'PORT': os.getenv('PSQL_PORT'),
+    },
+    'dados': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': os.getenv('PSQL_DB'),
+        'USER': os.getenv('PSQL_USER'),
+        'PASSWORD': os.getenv('PSQL_PASSWORD'),
+        'HOST': os.getenv('PSQL_HOST'),
+        'PORT': os.getenv('PSQL_PORT'),
+    },
+    'forecast': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': os.getenv('PSQL_DB'),
+        'USER': os.getenv('PSQL_USER'),
+        'PASSWORD': os.getenv('PSQL_PASSWORD'),
+        'HOST': os.getenv('PSQL_HOST'),
+        'PORT': os.getenv('PSQL_PORT'),
+    },
 }
 
-MEMCACHED_HOST = config('MEMCACHED_HOST', '127.0.0.1')
-MEMCACHED_PORT = config('MEMCACHED_PORT', '11211')
-QUERY_CACHE_TIMEOUT = config('QUERY_CACHE_TIMEOUT', 60 * 60, cast=int)
+PSQL_DB = os.getenv('PSQL_DB')
```

Comment:
@esloch I think it would be better to move this code (line 144-148) to line 116 ...

so you can use it instead of os.getenv each time in the lines between 118 and 141

Reply:
Reconfigurado e testado com make deploy e make deploy_staging, vou commitar e fazer um novo push:

![db_conn](https://user-images.githubusercontent.com/3450741/70003476-fd014d80-1541-11ea-91d6-13a6b5de630e.png)

Thanks @xmnlab!